apps/signing/serializers.py

```diff
@@ -16,6 +16,7 @@ class FileUploadSerializer(serializers.ModelSerializer):
     passed_review = serializers.SerializerMethodField('get_passed_review')
     processed = serializers.BooleanField(source='processed')
     reviewed = serializers.SerializerMethodField('get_reviewed')
+    valid = serializers.BooleanField(source='passed_all_validations')
```

Comment:
is `valid` needed? It's confusing to me because the `FileUpload` model already has a `valid` field and this looks different. If so, maybe it can be renamed?